src/overview/pricing/sponsored.md

```diff
@@ -0,0 +1,40 @@
+* An online non-profit community project that supports one of the above.
+
+The Platform.sh Developer Relations team manages the program and collectively has final say on what projects are eligible for sponsorship.
+
+Platform.sh will periodically reevaluate sponsored sites to ensure they remain eligible, and may with a month notice terminate the free service at its own will. Platform.sh may terminate this policy providing a month notice to allow users to migrate away
+
+## Offering
+
+An eligible project is entitled to one (1) Standard instance, with the standard 3 environments and 5 GB of storage.  Any number of users may be added as needed for the project at no additional cost.  The site is to be used exclusively for content supporting and promoting the project.  (Brochureware site, documentation, user forums, etc. are all acceptable.)
+
+There is no limit on the number of domains or application containers (subject to resource limits of the Standard Plan) that a project may use, although the project is expected to register and manage its own domain(s). Platform.sh will provision SSL certificates for free (as may be limited by the Let’s Encrypt API) but the hosted site may provide their own.
+
+A project is under no obligation to use the application container of their project.  (Eg, a PHP project is free to use Jekyll to produce a static site using Platform.sh, a Python project is free to host a PHPBB site, etc.)
+
+## Expectations
+
+In return, eligible projects are expected to include an HTML snippet conspicuously on each page of the site.  That may be in a sidebar or footer, with text no smaller than the standard body text size for the site. The image may be scaled via CSS as appropriate as long as the text is fully readable.  The badge markup will be provided by Platform.sh when setting up the site and will include a link to Platform.sh and an analytics tracking value so we know where links are coming from.  It does not include any cookies.
+
+Platform.sh may update from time to time the widget’s contents and the sponsored site will update the contents in a timely fashion.
+
+The sponsored site may ask, and Platform.sh shall accept variations of the widget as long as these fall within the branding guidelines of Platform.sh.
+
+Platform.sh may announce on its own media presence both online and offline that it hosts the said project, and kindly ask the sponsored site communicate at least once over its usual social media presence (Twitter, Mailing list, etc) the sponsored hosting.
+
+If the eligible project is a registered non-profit organization in the US or Germany, it is also expected to provide paperwork to register an in-kind donation to the organization of the equivalent price of the service ($50 USD/month or the EUR equivalent), for tax purposes.
+
+Sponsored projects will receive the same level of support as any other self-service hosted customer site.
+
+## Contact
```

Comment:
missing text?

Reply:
Waiting on the form from Chris Yates so we can embed a button for it.